src/status_im/ui/screens/browser/open_dapp/views.cljs

```diff
@@ -32,7 +32,7 @@
                 [vector-icons/icon :main-icons/browser {:color colors/gray}]]}]])
 
 (def dapp-image-data {:image (:dapp-store resources/ui) :width 768 :height 333})
-(def dapp-image (components.common/image-contain nil dapp-image-data))
+(def dapp-image [components.common/image-contain nil dapp-image-data])
```

Comment:
it was intentional because this is a constant we don't need to call it each time

Reply:
btw maybe we could consider this optimization in other places in the app

Reply:
or it will take more time on startup? hm

Reply:
well that's just for testing: if it is called right there subscription is also initialized right away which causes exception at the moment